TestFCWTAPI/CWTObjectTests.cs

```diff
@@ -49,6 +47,11 @@ public static void TestSplitRealAndImaginary()
             cosineCWT.SplitRealAndImaginary(CWTObject.CWTComponent.Both, out double[,] realCwt, out double[,] imagCwt);
             Assert.AreEqual(imagCwt[0, 5], cosineCWT.OutputCWT[1, 5]);
             Assert.AreEqual(realCwt[0, 21], cosineCWT.OutputCWT[0, 21]);
+            cosineCWT.SplitRealAndImaginary(CWTObject.CWTComponent.Imaginary, out double[,] imPlaceHolder, out double[,] imagOnlyCWT);
```

Comment:
Review my pull request (https://github.com/avcarr2/FCWTNET/pull/21) and then these tests will probably need to be updated.

Reply:
All unit tests have been adapted to pass with the changes from #21.